Dockerfile

```
@@ -41,6 +46,13 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 COPY --from=builder /root/kms/target/release/cosmian_kms_server /usr/bin/cosmian_kms_server
+COPY --from=builder /root/kms/target/release/ckms               /usr/bin/ckms
+
+#
+# Create working directory
+#
+RUN mkdir /data
+WORKDIR /data
```

`WORKDIR` cmd already creates the folder if it doesn't exist
```suggestion
WORKDIR /data
```

---

right